projects/igniteui-angular/src/lib/grids/column-actions/column-actions.component.ts

```diff
@@ -62,7 +62,7 @@ export class IgxColumnActionsComponent implements DoCheck {
      * this.columnHidingUI.cssClass = 'column-chooser';
      * ```
      */
-    @HostBinding('attr.class')
+    @HostBinding('class')
```

Comment:
Would also be nice if someone can tell me if/why this component is public API

Reply:
`cssClass` property is public in most of our components(i.e., badge, avatar, banner). Looks to me like a forgotten `@hidden @internal` tags and documented by mistake.

Reply:
Oh just noticed this is actually not hidden/internal. I only replaced the binding, because I can't figure out the deal with the component itself - it's in the public API, but no mention/usage in the docs/samples I can see. Just two grid dev demos.

Anyways, I could just mark this particular binding as private like the rest if you'd like. The other components and refactoring their host bindings to be private - that goes bit out of scope for this PR I think, but could be done separately  for sure.